.pylintrc

```diff
@@ -394,7 +394,8 @@ timeout-methods=requests.api.delete,requests.api.get,requests.api.head,requests.
 [EXCEPTIONS]
 
 # Exceptions that will emit a warning when caught.
-overgeneral-exceptions=Exception
+overgeneral-exceptions=builtins.BaseException,
```

Comment:
How come this has changed?.

Reply:
Right ya, those are only warning that it will eventually break. Happy for it to stay given it is there now. It would be good to call those or or just raise a quick PR of its own.